README.md

```diff
@@ -3,6 +3,9 @@
 
 ## Nuget packages
 
+- For ILogger:
+ [Microsoft.ApplicationInsights.NLogTarget](https://www.nuget.org/packages/Microsoft.Extensions.Logging.ApplicationInsights/)
```

Comment:
The name is wrong, should be [Microsoft.Extensions.Logging.ApplicationInsights]

Reply:
thnx for pointing out.!